src/EnergyPlus/HVACUnitaryBypassVAV.cc

```diff
@@ -213,31 +213,31 @@ namespace HVACUnitaryBypassVAV {
         // METHODOLOGY EMPLOYED:
         // Calls ControlCBVAVOutput to obtain the desired unit output
 
-        QSensUnitOut = 0.0;  // probably don't need this initialization
-        Real64 HeatingPower; // Power consumption of DX heating coil or electric heating coil [W]
+        QSensUnitOut = 0.0; // probably don't need this initialization
 
-        auto &CBVAV(state.dataHVACUnitaryBypassVAV->CBVAV);
+        auto &changeOverByPassVAV = state.dataHVACUnitaryBypassVAV->CBVAV(CBVAVNum);
```

Comment:
Applied ShotCuts `auto &changeOverByPassVAV = state.dataHVACUnitaryBypassVAV->CBVAV(CBVAVNum)` in `SimCBVAV()` function.